### ISSUE TITLE
As a person with a disability, I want to use Rabble Rouser just as well as anyone else

### ISSUE THREAD

**Issue by camjackson** [rabblerouser/core#118]
Aka accessibility.

Some links to help:

 - Talk: [React loves accessibility (and it's not difficult to add)](https://www.youtube.com/watch?v=L6DgX0EbUAU)
 - Tutorial: http://accessibility.parseapp.com/
 - Tutorial: https://teachaccess.github.io/tutorial/
 - [Webaim](http://webaim.org/articles/), especially [keyboard navigation](http://webaim.org/techniques/keyboard/)

Stuff we can/should do:

- [x] Use eslint-plugin-jsx-a11y
- [ ] Use [react-a11y](https://github.com/reactjs/react-a11y)
- [ ] Maybe use Paypal's [Automated Accessibility Testing Tool](https://github.com/paypal/AATT)?
- [ ] Test with [WAVE](http://wave.webaim.org/extension/)?
- [ ] Ensure the whole app can be navigated with a keyboard
- [ ] Ensure text colour contrast gets a AAA rating from [here](http://juicystudio.com/services/luminositycontrastratio.php#specify)
- [ ] Test the signup app with a screen-reader / with a person for whom accessibility is important
- [ ] Test the admin app with a screen-reader / with a person for whom accessibility is important

**Comment by camjackson** [comment]
@Letitia-May you current gig has especially high accessibility requirements, right? Anything extra we could/should be doing here?

**Comment by Letitia-May** [comment]
@camjackson a lot of what we've looked at on the current project is covered in the list here.

Mac laptops come with an inbuilt screenreader called VoiceOver, which is good for testing. Webaim has useful tips for [keyboard navigation](http://webaim.org/techniques/keyboard/) and other areas of accessibility (http://webaim.org/articles/). Also, where there is text on a coloured background, the contrast of the text colour and background colour should pass at a level of AAA on [this site](http://juicystudio.com/services/luminositycontrastratio.php#specify).

The gov.uk site is a leader in accessibility standards.

**Comment by camjackson** [comment]
Great, thanks. I've updated the issue :)
